mhs/outbound/outbound/request/synchronous/tests/test_handler.py

```diff
@@ -309,3 +311,49 @@ def setup_workflows(self):
 
         self.sync_async_workflow.handle_sync_async_outbound_message.return_value = result
         self.workflow.handle_outbound_message.return_value = test_utilities.awaitable((200, "Success"))
+
+
+class TestSynchronousHandlerSyncMessage(tornado.testing.AsyncHTTPTestCase):
```

Comment:
Does this need to be a separate test class?

Reply:
Possibly not, I set it as a different class because the magic mock for the 'default' workflow I needed it to inherit from sync workflow to do that `isinstance` check, I didn't really want to enforce that type on the other tests so I figured it should be a separate class